react/react1/week2/react1-week2-app/src/components/TodoTask.js

```diff
@@ -0,0 +1,30 @@
+function TodoTask(props) {
+  // id in li.id is exposed to later target it with remover. span.id is exposed for visual clarity for testing purposes.
+  return (
+    <li className="todoItemCard" id={props.id}>
+      <div>
+        <p className="taskText"> Task : {props.description} </p>
+        <span> ID = {props.id}</span>
+      </div>
+      <div className="cardControlls">
+        <label class="switch">
+          <input type={"checkbox"} onChangeCapture={handleChange}></input>
+          <span class="slider round"></span>
+        </label>
+        <button id="deleteTask" onClick={props.removeTodo}>
```

Comment:
Here's another way to do this, and then use the todo id to filter the todos list instead of event.target...

```
<button id="deleteTask" onClick={() => props.removeTodo(todo.id)} >
```